data/marauder variants.txt

```diff
@@ -0,0 +1,317 @@
+# Copyright (c) 2014 by Michael Zahniser
```

Comment:
[I just skimmed through it, if i have time i will put a full review later.]

Unless this gets moved to an existing file, this header should reference you, since it's entirely your work - and even then, if the contributions are significant. See [this comment](https://github.com/endless-sky/endless-sky/pull/3959/files#r226786785) for reasoning. Also, the `remnant missions.txt` header, after zitchas' additions in #3761:
https://github.com/endless-sky/endless-sky/blob/95d7cf5d3c2bf88cd065f4697b0bca670ccb27ba/data/remnant%20missions.txt#L1-L3